vsintegration/src/vs/FsPkgs/FSharp.Project/FS/MSBuildUtilities.fs

```diff
@@ -12,6 +12,60 @@ open System.Diagnostics
 open System.IO
 open System
 
+module internal MSBuildHelpers =
+
+    let FolderComparer = StringComparer.OrdinalIgnoreCase 
+
+    let GetPathParts (x:string) =
+        x.Split( [| '\\' |], StringSplitOptions.RemoveEmptyEntries) |> Seq.ofArray
+
+    let IsSameDir a b =
+        0 = Seq.compareWith (fun x y -> FolderComparer.Compare(x,y)) (GetPathParts a) (GetPathParts b)
+
+    let IsSubDirOf baseDir directory  =
```

Comment:
If they are always used together, they can still be combined and tested as a single function.